app/controllers/levels_controller.rb

```diff
@@ -233,8 +244,10 @@ def step_query_exists?
     end
 
     def get_matched_data(input)
-      pattern = /([A-Z][a-z]*)\.([a-z]*_?[a-z]*)\(?{?(:?[a-z]*_?[a-z]*):?(=?>?)(\w*)}?\)?\.?([a-z]*)\(?(\d?)\)?/
-      input.match(pattern).to_a.drop(1)
+      # pattern = /([A-Z][a-z]*)\.([a-z]*_?[a-z]*)\(?{?(:?[a-z]*_?[a-z]*):?(=?>?)(\w*)}?\)?\.?([a-z]*)\(?(\d?)\)?/
+      # input.match(pattern).to_a.drop(1)
+      pattern = /([a-zA-Z]+)(=?)([A-Z]*[a-z]*)\.([a-z]*_?[a-z]*)\(?{?(:?[a-z]*_?[a-z]*):?(=?>?)(\w*)}?\)?\.?([a-z]*)\(?(\d?)\)?/
```

Comment:
Update pattern to match variables, since I added the Step queries. Previously, this only matched the class name that had a capital letter.